site/operational-guide/tasks.md

```diff
@@ -2,26 +2,37 @@
 title: Administrative Tasks
 menu_order: 60
 ---
-##<a name="start-on-boot"></a>Configure Weave to Start Automatically on Boot
 
-`weave launch` runs all weave's containers with a Docker restart
-policy of `always`, so as long as you have launched weave manually
-once and your system is configured to start Docker on boot then weave
-will be started automatically on system restarts.
+The following administrative tasks are discussed: 
 
-If you're aiming for a non-interactive installation, you can use
-systemd to launch weave after Docker - see [systemd
+* [Configuring Weave to Start Automatically on Boot](#start-on-boot)
+* [Detecting and Reclaiming Lost IP Address Space](#detect-reclaim-ipam)
+* [Upgrading a Cluster](#cluster-upgrade)
+* [Resetting Persisted Data](#reset)
+
+
+##<a name="start-on-boot"></a>Configuring Weave to Start Automatically on Boot
+
+`weave launch` runs all of Weave's containers with a Docker restart
+policy set to `always`.  If you have launched Weave manually at least
+once and your system is configured to start Docker on boot, then Weave
+will start automatically on system restarts.
+
+If you are aiming for a non-interactive installation, use
+systemd to launch Weave after Docker - see [systemd
```

Comment:
This should probably say "systemd or a similar init system", and note that weave should be launched with `--no-restart` (that's right, isn't it, @awh?)